### PR TITLE
Add deck mode

### LINKED ISSUE #77: Connect Library and Deck methods to LogicManager
Linked to #66, #59

### LINKED ISSUE #78: Add functionality to switch from Library view to Deck view, and vice versa

### PR DESCRIPTION
**Pull request by kschiew**
Closes #77 #78

### CODE REVIEW

File: src/main/java/seedu/address/logic/parser/LibraryParser.java [hunk 1]
```diff
@@ -52,17 +46,17 @@ public Command parseCommand(String userInput) throws ParseException {
...
+            return new CreateDeckCommandParser().parse(arguments);
 
-        case EditCommand.COMMAND_WORD:
-            return new EditCommandParser().parse(arguments);
+        case DeleteDeckCommand.COMMAND_WORD:
+            return new DeleteDeckCommandParser().parse(arguments);
 
-        case DeleteCommand.COMMAND_WORD:
-            return new DeleteCommandParser().parse(arguments);
+        case ClearDeckCommand.COMMAND_WORD:
```
Comment (ncslzh) [inline src/main/java/seedu/address/logic/parser/LibraryParser.java]: can be removed since feature is dropped

**Comment by ncslzh** [comment]
ClearDeckCommand and ClearDeckParser can be removed

File: src/main/java/seedu/address/logic/parser/LibraryParser.java (hunk 1 above)
Comment (kschiew) [reply src/main/java/seedu/address/logic/parser/LibraryParser.java]: Done!

**Comment by amrl** [comment]
LGTM 💯

**Pull request merged**